content/new/differences.md

```diff
@@ -1,5 +1,52 @@
 # Differences from older platform 
 
+## Network
+In the new platform, there is 3 networks to choose from (attach only one network):
+
+1. public: This network will give you a public v4 ip address, public v6 address, dns setup and default gateway so it is reachable directly to/from Internet.
+2. default: This network will give you a private ip on a RFC 1918 network,
+   dns setup and default gateway with Network Address Translation (NAT) for outgoing traffic so instances can reach 
+   services on the Internet, in addtion to instances on other networks in Safespring Compute (provided it is allowed by means of security groups).
```

Comment:
ADD: Before the v2 platform reaches production the default network will ALSO provide a public v6 address. This means you can have public IPv6 for free, as the public network will be priced per IPv4 address.